terraform/application/config/production.tfvars.json

```diff
@@ -0,0 +1,11 @@
+{
+    "cluster": "production",
+    "namespace": "tra-production",
+    "evidence_container_retention_in_days": 30,
```

Comment:
TBC by developers

Reply:
Have removed for time being

Reply:
Developers have agreed with 30 days, this can be set